src_common/hash_bytes.c

```diff
@@ -9,7 +7,7 @@
  * @param[in] n number of bytes to hash
  * @param[in] hash_ctx pointer to the hashing context
  */
-void hash_nbytes(const uint8_t *const bytes_ptr, uint8_t n, cx_hash_t *const hash_ctx) {
+void hash_nbytes(const uint8_t *bytes_ptr, uint8_t n, cx_hash_t *hash_ctx) {
```

Comment:
`feed_nbytes_to_hash()` ?

Reply:
Can be looked at in another PR, since these functions predate this PR and are used elsewhere in the app.